auth-web/public/config/configuration.json

```diff
@@ -1,6 +1,6 @@
 {
   "VUE_APP_PAY_ROOT_API": "https://pay-api-dev.pathfinder.gov.bc.ca/api/v1",
-  "VUE_APP_AUTH_ROOT_API": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1",
+  "VUE_APP_AUTH_ROOT_API": "http://localhost:5000/api/v1",
```

Comment:
Can we keep the default config to point to dev? Some of the designers, etc. don't run the API locally and this will trip them up.

Reply:
oops, Sorry about that, will correct this. Thank you @sutherlanda :)